tools/system_libs.py

```diff
@@ -710,7 +712,6 @@ def get_depends(self):
 
 class libc_wasm(MuslInternalLibrary):
   name = 'libc-wasm'
-  cflags = ['-O2', '-fno-builtin']
```

Comment:
this part deeply worries me - sin/cos/etc. may not get inlining with `-Os`. Let's keep `-O2` here unless we benchmark carefully and are confident it's ok?